tests/e2e/contacts/new-lineage-structure.wdio-spec.js

```diff
@@ -0,0 +1,43 @@
+const contactPage = require('../../page-objects/contacts/contacts.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const utils = require('../../utils');
+const faker = require('faker');
+
+const centerName = faker.address.city();
+const centerContact = faker.name.findName();
+const area = faker.address.city();
+const areaContact = faker.name.findName();
+const household = faker.address.city();
+const householdContact = faker.name.findName();
+
+
+describe('Create new lineage structure', () => {
+  before(async () => {
+    await loginPage.cookieLogin();
+  });
+
+  after(async () => {
```

Comment:
It would be real cool if we could find a wdio or mocha hook that does this for us and not have to add it everywhere.

Reply:
will have a look at this as well. we also need a hook or hooks for before and/or beforeEach